client/src/pages/private/ParticipantView.js

```diff
@@ -131,6 +134,8 @@ export default () => {
   const [locations, setLocations] = useState([]);
 
   const handleRequestSort = (event, property) => {
+    if (property === 'freshness') property = 'updated_at';
```

Comment:
Try to avoid modifying function parameters. I know this is a primitive type and doesn’t mutate the original string but in case of an object this could lead to a code smell.

Reply:
yea, this bit was kind of slapped on as a band-aid - I'll fix it up.